feature-configs/deploy/metallb/operator_catalogsource.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: metallb-operator
+  namespace: metallb-system
+spec:
+  displayName: MetalLB Operator Upstream
+  # Change this to registry.redhat.io/... once metallb operator is released
+  image: quay.io/mmirecki/metallb-operator-index
```

Comment:
to what operator version this index point?

Is the openshift/metallb-operator or the u/s one?

Reply:
when the operator is released i think we should update the subscription to use the redhat-operators catalogsource and delete this catalogsource (like we do for ptp and sriov)

Reply:
Yes, this is of couse onlyl temporary.
This pointed to the latest master some time ago, I'll update that to latest when merged.

Reply:
can't we use the one @sabinaaledort you push to the operator hub? it should be available as part of the 
```
openshift-marketplace   community-operators             Community Operators                     grpc   Red Hat     20d
```
no?

Reply:
yes, we can definitely use the olm catalogsource.